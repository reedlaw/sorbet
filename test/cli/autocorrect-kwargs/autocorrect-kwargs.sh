tmp="$(mktemp)"
file="test/cli/autocorrect-kwargs/autocorrect-kwargs.rb"
cp "$file" "$tmp"
main/sorbet --silence-dev-message -a "$tmp"
diff "$file" "$tmp"
rm "$tmp"
