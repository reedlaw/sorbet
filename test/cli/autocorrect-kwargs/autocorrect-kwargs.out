6c6
< sig {returns(foo: Integer, bar: String)}
---
> sig {returns({foo: Integer, bar: String})}
13c13
< sig {returns(T::Array[foo: Integer])}
---
> sig {returns(T::Array[{foo: Integer}])}
18c18
< foo = T::Array[foo: Integer, bar: String].new
---
> foo = T::Array[{foo: Integer, bar: String}].new
21c21
< sig {params({a: Integer, b: String}).void}
---
> sig {params(a: Integer, b: String).void}
